schemas/protocol.json

```diff
@@ -2,12 +2,15 @@
   "$schema": "https://json-schema.org/draft/2020-12/schema",
   "title": "Message",
   "description": "The messages supported by the StepFlow protocol. These correspond to JSON-RPC 2.0 messages.\n\nNote that this defines a superset containing both client-sent and server-sent messages.",
-  "anyOf": [
+  "oneOf": [
```

Comment:
Changed this to render a little better. Other updates were because of the missed tests allowing drift.